moztelemetry/partition-packing/PartitionMainPings.py

```diff
@@ -0,0 +1,26 @@
+# This program is a benchmark test for the partition behavior of running Spark Jobs on a cluster. It uses 6 
+# Paritions and the data for Main Pings.
+
+from datetime import datetime 
+from pyspark import SparkContext 
+ 
+from moztelemetry.dataset import Dataset 
+ 
+if __name__ == '__main__': 
+  startTime = datetime.now() 
+  sc = SparkContext("local", "ecole") 
+  numPartitions = 6 # specification for partition behavior 
```

Comment:
If doctype or channel is specified as an argument, the number of partitions will likely change. Keeping the number of partitions consistent is important for interpreting the results of the benchmark.